app/services/MemberSearchService.scala

```diff
@@ -116,6 +109,43 @@ class MemberSearchService @Inject()(
     }
     allRows.sortBy(_.name)
   }
+
+  def jsonSearch: (String, JsValue) => Option[JsValue] = (searchString, ua) => {
```

Comment:
I think this can be a val. If possible it would be better to make it private (maybe package private as it needs to be accessible from unit tests).